lib/player.js

```diff
@@ -3596,6 +3596,24 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     return ret;
   }
 
+  /**
+   * Return a copy of the current non default configuration.  Modifications of
+   * the returned value will not affect the Player's active configuration.
+   * You must call <code>player.configure()</code> to make changes.
+   *
+   * @return {!Object}
+   * @export
+   */
+  getNonDefaultConfiguration() {
+    goog.asserts.assert(this.config_, 'Config must not be null!');
+
+    const ret = this.defaultConfig_();
+    shaka.util.PlayerConfiguration.mergeConfigObjects(
```

Comment:
I don't see why this merge step is necessary.  this.config_ should already follow the proper format, right?  In fact, you don't even use the result `ret` here at all.